Docs update (#227)
nf-cmgg/germline: CHANGELOG.md

1. Validation of all samples now use an intersect of the golden truth BED files with the BED file used to call the variants. This should fix the WES validation which was broken until this point.
2. A couple of small fixes to the vardict flow.
3. Only use the standard chromosomes for UPDio analysis.
4. Reduced the resources given to some GATK4 modules

> 5. VCF2DB now uses a seqera container to fix some issues when running it in nomad

Reviewer: ```suggestion
5. VCF2DB now uses a seqera container to fix some issues when running with `-profile docker`
```
